src/ecs/audio/bundle.rs

```diff
@@ -21,14 +21,15 @@ use shred::ResourceId;
 ///
 /// Panics during `DjSystem` registration if the bundle is applied twice.
 ///
-pub struct DjBundle<'a> {
+pub struct DjBundle<'a, F> {
```

Comment:
Since Dj is gone, maybe rename to AudioBundle?

Reply:
But there's still `DjSystem`.